proto/zetachain/zetacore/crosschain/tx.proto

```diff
@@ -149,6 +149,14 @@ message MsgVoteOutbound {
   pkg.coin.CoinType coin_type = 9;
 }
 
+// InboundStatus represents the status of an observed inbound
+enum InboundStatus {
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  success = 0;
+  insufficient_depositor_fee = 1;
```

Comment:
Can we add comment and precise this is specifically for Bitcoin

Reply:
updated https://github.com/zeta-chain/node/pull/3358/commits/a0604ef8e5e77ee5253aaa3a3775388b36496a6f